modules/ossm-about-revisionbased-strategy.adoc

```diff
@@ -0,0 +1,6 @@
+:_mod-docs-content-type: Concept
+[id="about-revisionbased-strategy"]
+= About RevisionBased strategy
+:context: ossm-about-revisionbased-strategy
+
```

Comment:
During updates, the `RevisionBased` strategy typically runs two revisions of a control plane. This capability allows you to gradually migrate workloads from the old control plane to the new control plane, making canary upgrades possible. This update strategy also allows you to upgrade by more than one minor version.